manufacturing/blower/README.md

```diff
@@ -92,3 +92,20 @@ You will have to crimp connectors to the second end of the wiring that connects
 to the [PCB](../../pcb).
 It depends on your build layout, but in the case that you need longer wires, you will have to crimp both ends.
 In such a case you might need a greater number of XH and VH connector components listed in the BOM above.
+
+## Assembly Instructions of Blower to the Enclosure
+
```

Comment:
I do not think this is the correct sequence for assembling. It must start with the driver.